buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/CopyRestApiTask.java

```diff
@@ -109,7 +111,8 @@ public FileTree getInputDir() {
 
     @OutputDirectory
     public File getOutputDir() {
-        return new File(getTestSourceSet().getOutput().getResourcesDir(), REST_API_PREFIX);
+        assert Util.getTestSourceSet(getProject()).isPresent();
```

Comment:
We should use `orElseThrow()` here and supply an exception with an appropriate message.

Reply:
I added a message to the `assert`